docs/.vuepress/styles/palette.styl

```diff
@@ -0,0 +1,13 @@
+$accentColor = #1847de
```

Comment:
File name is on purpose ?

Reply:
Yeah that's a Stylus file, this is how you customize variables in Vuepress